src/pages/signin/LoginForm.js

```diff
@@ -111,6 +111,11 @@ class LoginForm extends React.Component {
         if (this.props.account.errors) {
             Session.clearAccountMessages();
         }
+
+        // Clear the "Account successfully closed" message when the user starts typing
+        if (this.props.closeAccount.success) {
+            Onyx.merge(ONYXKEYS.FORMS.CLOSE_ACCOUNT_FORM, CONST.DEFAULT_CLOSE_ACCOUNT_DATA);
```

Comment:
@Gonals Onyx is not defined and linting says we can't use `Onyx.merge` outside of actions, can we add this to an action?

Reply:
Fixed!